tensorflow-mnist-input-data/Setup.hs

```diff
@@ -83,25 +88,25 @@ verify filePath hash = do
                  ++ helpfulMessage
               )
 
-urlPrefix = "http://yann.lecun.com/exdb/mnist/"
+urlPrefix = "http://web.archive.org/web/20170303140123/http://yann.lecun.com/exdb/mnist/"
 
 -- | File names relative to 'urlPrefix' and their sha256.
 fileInfos = [
-    ( "train-images-idx3-ubyte.gz"
```

Comment:
I changed it to be in alphanumeric order so it's train...1 train...3 t10k...1 t10k...3 instead of 3-1-3-1.

This tends to be default ordering when listing files. For example when I was matching sha hashes at the command line, it was easier to keep the mapping straight.

It's not a huge deal, I can change it back if there's a better reason to keep it as is.